core/src/qos_service.rs

```diff
@@ -49,6 +51,7 @@ pub struct QosService {
     // metrics reporting runs on a private thread
     reporting_thread: Option<JoinHandle<()>>,
     running_flag: Arc<AtomicBool>,
+    pub bidirection_reply_service: QuicBidirectionalReplyService,
```

Comment:
none of it's members are pub, most services structs are strictly using private fields